stl/inc/xutility

```diff
@@ -5544,17 +5544,35 @@ _NODISCARD constexpr auto lexicographical_compare_three_way(
 }
 #endif // __cpp_lib_concepts
 
+template <class _Iter, class _Ty, class _Elem = _Iter_value_t<_Iter>>
+_INLINE_VAR constexpr bool _Vector_alg_in_find_is_safe = // Can we activate the vector algorithms for find/count?
+    _Iterator_is_contiguous<_Iter> // The iterator must be contiguous so we can get raw pointers.
+    && !_Iterator_is_volatile<_Iter> // The iterator must not be volatile.
+    && disjunction_v< // And one of the following conditions must be met:
+#ifdef __cpp_lib_byte
+        conjunction<is_same<_Ty, byte>, is_same<_Elem, byte>>, // We're finding a std::byte in a range of std::byte.
+#endif // __cpp_lib_byte
+        conjunction<is_integral<_Ty>, is_integral<_Elem>>, // We're finding an integer in a range of integers.
+        // The integer types can be different, which requires careful handling.
+        conjunction<is_pointer<_Ty>, is_same<_Ty, _Elem>>, // We're finding a U* in a range of U* (identical types).
+        conjunction<is_null_pointer<_Ty>, is_pointer<_Elem>>>; // We're finding a nullptr in a range of U*
+
+
 template <class _InIt, class _Ty>
-_NODISCARD constexpr bool _Within_limits(const _InIt&, const _Ty& _Val) {
+_NODISCARD constexpr bool _Within_limits(const _Ty& _Val) {

```

Comment:
On Discord, @CaseyCarter observed:

> `_Within_limits` seems like a bit of a misnomer, doubly so considering the extension to `nullptr`. [...] `_Could_possibly_compare_equal` is a bit long, but precise.

I agree that a rename would be appropriate (and there's already a signature change). `_Could_compare_equal` might be sufficient ("possibly" doesn't really add anything AFAICT).